docs/local_resource.md

```diff
@@ -0,0 +1,100 @@
+you'd rather compile your binary locally, and then use Live Update to quickly sync
+the new binary up to a running container in your cluster
+
+See also: [`local_resource` API spec](api.html#api.local_resource).
+
+(Of course, the command you run doesn't have to be on that (only) affects your
+local filesystem. It might, for instance, be a script that is _invoked locally_
+but runs against your k8s cluster.)
+
+## Local Resource + Live Update = <3
+
+Local Resource and Live Update are two great tastes that taste great together.
+A really useful pattern is to have a Local Resource which creates an artifact
+that is then picked up by a Live Update and synced to a running container.
+For example, here's how you might use this combination to generate protobufs:
+```python
+# whenever `helloworld.proto` changes, generate Python protobufs
+local_resource('proto',
+    cmd='protoc --proto_path=src --python_out=build/gen',
+    deps=['helloworld.proto']
+)
+
+# define your hello-world resource: k8s yaml + a Docker build
+k8s_yaml('hello-world.yaml')
+docker_build('hello-world', '.',
+    ignore='helloworld.proto',  # don't step on Local Resource's toes
+    live_update=[ sync('.', '/app') ]
+)
+```
+
+Note that the `docker_build` call specifies `ignore='helloworld.proto'`. This is
+because we DON'T want an edit to that proto file to kick off a Docker build.
+Rather, an edit to `helloworld.proto` triggers the Local Resource to generate
+protobufs; when these protobufs appear on disk, they register as file changes and
+trigger a Live Update to the `hello-world` resource, i.e. they get `sync`'d to
+the container where `hello-world` is running.
+
+To see this pattern in action, check out [this example repo](https://github.com/windmilleng/local_resource_example).
+
+# Run occasional workflows (locally or against your cluster)
+
+There's another class of workflow that Local Resource can help with; commands that
+are a part of your development flow, but that you run only occasionally. This might
+be a task like "ensure GKE user 'foo' exists" or "refresh my credentials" or even
+"blow away my database".
+
+You can trigger a Local Resource at will with the "force update" button:
+
+!["force update" button](assets/img/force-update-button.png)
+
+By default, a Local Resource will run on startup. To disable this behavior, put the
+resource in `TRIGGER_MODE_MANUAL` and specify `auto_init=False`:
+```python
+local_resource('reset-db', cmd='reset_db.sh',
+    trigger_mode=TRIGGER_MODE_MANUAL, auto_init=False
+)
+```
+
+(`auto_init=False` is currently only compatible with `TRIGGER_MODE_MANUAL`. If
+you'd like a Local Resource that runs automatically in response to file changes
+but does NOT run on `tilt up`, [let us know](https://tilt.dev/contact).)
+
+Note that you can mix and match manual and automatic runs as you like. For instance,
+you might have a `seed-db` Local Resource. Usually, you run it manually whenever
```

Comment:
this seems like a weird solution to this use case, since seed_db.sh would run on startup